detoks/src/main/java/nl/tudelft/trustchain/detoks/DeToksCommunity.kt

```diff
@@ -2,13 +2,14 @@ package nl.tudelft.trustchain.detoks
 
 import android.content.Context
 import android.util.Log
-import nl.tudelft.ipv8.Community
 import nl.tudelft.ipv8.Overlay
 import nl.tudelft.ipv8.Peer
 import nl.tudelft.ipv8.messaging.Packet
 
 
-class DeToksCommunity(private val context: Context) : Community() {
+class DeToksCommunity(
+    private val context: Context
+    ) : TransactionEngine("c86a7db45eb3563ae047639817baec4db2bc7c25") {
```

Comment:
I understand that serviceId was hardcoded in the code but it seems to me that it should not be, maybe use some environment file or something to hide it?